package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
```

Comment:
We should add a minimum node version here. Not all the node versions support the features like string templates or es6 exports.

Reply:
Added this to a task.